samples/standalone/restdocs/http-server/build.gradle

```diff
@@ -26,17 +17,10 @@ repositories {
 	maven { url "https://repo.spring.io/release" }
 }
 
-apply plugin: 'groovy'
-apply plugin: 'org.springframework.boot'
-apply plugin: 'io.spring.dependency-management'
-apply plugin: 'maven-publish'
-apply plugin: 'maven'
-apply plugin: 'spring-cloud-contract'
-
 dependencyManagement {
 	imports {
-		mavenBom "org.springframework.cloud:spring-cloud-dependencies:$BOM_VERSION"
-		mavenBom "org.springframework.cloud:spring-cloud-contract-dependencies:${project.findProperty('verifierVersion') ?: verifierVersion}"
```

Comment:
Since I expect for this to come up probably... this elvis operator is actually redundant, so that's why I simplified it below.

If you pass a -P argument, it automatically overrides whatever is in your `gradle.properties` file with no further action necessary. So both `findProperty('verifierVersion')` and `verifierVersion` in this case are equivalent (and ironically, we can **never** hit the second case because verifierVersion is in `gradle.properties`). If verifierVersion did _not_ exist in our `gradle.properties` file, we would want to solely use `findProperty('verifierVersion')` (again without the elvis), because this gives us null safe protection for Gradle properties. **However**, when the context of the property's usage is taken into account then `null` is an invalid value, thus `verifierVersion` is much simplier and provides an simplier, more readable codebase.